partner_fax/models/res_company.py

```diff
@@ -0,0 +1,8 @@
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+
+    _inherit = 'res.company'
+
+    fax = fields.Char()
```

Comment:
I don't think this is needed. make a related field to partner_id like we have phone and email.
Save redundancy.

Reply:
I think this is the same opinion as @bealdav said in https://github.com/OCA/partner-contact/pull/572#issuecomment-370159649 . Isn't it? :thinking:

Reply:
yes, Similar, we can add field, but related type only.

Reply:
@satriani-vai thanks for this contribution, I think recommendation from @bealdav and @JayVora-SerpentCS is correct. Don't you think so?

Reply:
Yes it is. I would keep fax in all contact types.

Reply:
Hello @satriani-vai if you change this to ma a related field to partner_id, then @JayVora-SerpentCS and @bealdav could give you the :+1 for mergin :smiley: Could you make the change please? Thank you!

Reply:
@satriani-vai What happened bro?

We are still waiting for this.